core/array/append_spec.rb

```diff
@@ -1,35 +1,9 @@
 require File.expand_path('../../../spec_helper', __FILE__)
 require File.expand_path('../fixtures/classes', __FILE__)
+require File.expand_path('../shared/append', __FILE__)
 
-describe "Array#<<" do
-  it "pushes the object onto the end of the array" do
-    ([ 1, 2 ] << "c" << "d" << [ 3, 4 ]).should == [1, 2, "c", "d", [3, 4]]
+ruby_version_is '2.5' do
+  describe 'Array#append' do
+    it_behaves_like(:array_append, :append)
   end
```

Comment:
👍 
Hmm... But current push specs contains multiple arguments https://github.com/ruby/spec/pull/452/files#diff-3413327b97e53a57c0865d4cbe4929e9L7

I'll try to refine 💪